clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -432,7 +440,7 @@ void clearInFlightRequestCorrelationId() {
         inFlightRequestCorrelationId = NO_INFLIGHT_REQUEST_CORRELATION_ID;
     }
 
-    boolean hasInflightRequest() {
+    boolean hasInFlightRequest() {
```

Comment:
It does seem to be used more often, so it's fair to stick to that. It seems like there are a few uses of `Inflight` and `inflight` though. We can clean that up later though.